### ISSUE TITLE
Binding queue to exchange requires declaring exchange (creating exchange object)

### ISSUE THREAD

**Issue by jkr78** [mosquito/aio-pika#86]
Currently it is impossible to bind queue to exchange by name. 
I have to ether to declare exchange or create dummy exchange object or mock (object with name attribute). Redeclaring exchanges may cause errors if configuration changes on one end.

**Comment by mosquito** [comment]
Let's imagine how should it works. Should I add some additional methods e.g. `Queue.bind_by_name` and `Queue.unbind_by_name`?

**Comment by jkr78** [comment]
`*_by_name()` is fine.
Or break "Explicit is better than implicit" and have something like:
```python
def bind(self, exchange: Union[str, Exchange],
         routing_key: str=None, *, arguments=None, timeout: int = None) -> asyncio.Future:
    ...
    exchange_name = exchange.name if isinstance(exchange, Exchange) else exchange
    self._channel.queue_bind(
        f.set_result,
        self.name,
        exchange_name,
        routing_key=routing_key,
        arguments=arguments
    )
```

**Comment by decaz** [comment]
I suggest not to add any additional methods or break anything - just change `bind` behavior: accept exchange name instead of exchange object like it is implemented in pika. Caller will have to use `exchange.name` and that's all.

**Comment by mosquito** [comment]
@decaz your recipe implemented in `1.8.0`
